doc/ARCHITECTURE.md

```diff
@@ -100,10 +100,19 @@ AuthenticationSASLFinal and AuthenticationOk. The SSLRequest message is supporte
 
 The remote management interface is defined in [remote.h](../src/include/remote.h) ([remote.c](../src/libpgagroal/remote.c)).
 
-## libev usage
+## I/O layer
 
-[libev](http://software.schmorp.de/pkg/libev.html) is used to handle network interactions, which is "activated"
-upon an `EV_READ` event.
+Starting from version 2.0.0, the I/O layer interface is primarily defined in [ev.h](../src/include/ev.h) 
```

Comment:
You just need to explain the current situation - no version numbers necessary.

And `ev.h` **is defined**